umbraco/Umbraco.UI
# Change the storybook-vite-builder package to the official one

It looks like Storybook has taken over the package for the vite-builder:

https://www.npmjs.com/package/@storybook/builder-vite
https://github.com/storybookjs/builder-vite

I have updated our code to use the new dependency instead.


## How to test?

* Make sure storybook runs in dev mode
* Make sure the static build of storybook works



- [x] I confirm that to my knowledge the contribution looks original and that the [contributor is presumably allowed to share it](https://github.com/umbraco/Umbraco.UI/blob/dev/docs/CONTRIBUTING.md#ownership-and-copyright).
